packages/react-native-reanimated/scripts/validate-android.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-if grep -Rn android/src/main/cpp -e '#include "' ; then
-    echo "Found \`#include "..."\` in \`android/src/main/cpp/\` directory. Convert it to \`#include <...>\`, e.g. \`#include <reanimated/AndroidUIScheduler.h>\`." && false
+if grep -Rn android/src/main/cpp -e '^#include "'; then
+    echo "Found \`#include \"...\"\` in \`android/src/main/cpp/\` directory. Convert it to \`#include <...>\`, e.g. \`#include <reanimated/AndroidUIScheduler.h>\`." && false
```

Comment:
I think we can completely rid of `\` by using `'` instead of `"`:
```suggestion
    echo 'Found `#include "..."` in `android/src/main/cpp/` directory. Convert it to `#include <...>`, e.g. `#include <reanimated/AndroidUIScheduler.h>`.' && false
```